pkg/util/membercluster_ratelimiter.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2025 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"sync"
+
+	"k8s.io/client-go/util/flowcontrol"
+)
+
+var defaultRateLimiterGetter = &ClusterRateLimiterGetter{}
+
+// GetClusterRateLimiterGetter returns a ClusterRateLimiterGetter.
+func GetClusterRateLimiterGetter() *ClusterRateLimiterGetter {
+	return defaultRateLimiterGetter
+}
+
+// ClusterRateLimiterGetter is responsible for retrieving rate limiters for member clusters.
+// It dynamically creates and manages a singleton rate limiter for each member cluster,
+// using the preset default QPS and burst values to ensure consistent rate limiting across all clusters.
+type ClusterRateLimiterGetter struct {
+	limiters map[string]flowcontrol.RateLimiter
+	mu       sync.Mutex
+	qps      float32
+	burst    int
+}
+
+// SetDefaultLimits sets the default qps and burst for new rate limiters.
+// Do not call this method after calling GetRateLimiter, otherwise the call will not take effect.
+// If qps and limit are not set when using Getter, it will use 5 and 10 by default (the default qps and burst of client-go rest.Config).
```

Comment:
```suggestion
// If qps and limit are not set when using Getter, it will use 40 and 60 by default (the default qps and burst of '--cluster-api-qps' and '--cluster-api-burst' in karmada-controller-manager).
```
Shall we keep the default consistent with the flags?